### PR TITLE
[CHK-2851] fix exclude path double path

### PR DESCRIPTION
**Pull request by ciuffagianluca**
Fix deploy-pipeline

#### List of Changes

The double `exclude-path` option for sync command exclude the first one and undesired delete happens

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by vito80ba — APPROVED** [review]

File: .devops/pagopa-deploy-pipelines.yml [hunk 1]
```diff
@@ -117,7 +117,7 @@ stages:
               azureSubscription: 'DEV-PAGOPA-SERVICE-CONN'
               scriptLocation: inlineScript
               inlineScript: |
-                call az storage blob sync --container $web --account-name pagopadcheckoutsa -s "$(Pipeline.Workspace)\Bundle_DEV" --exclude-path payment-transactions-gateway --exclude-path ecommerce-fe
+                call az storage blob sync --container $web --account-name pagopadcheckoutsa -s "$(Pipeline.Workspace)\Bundle_DEV" --exclude-path payment-transactions-gateway ecommerce-fe
```
Comment (pietro-tota) [inline .devops/pagopa-deploy-pipelines.yml:120]:
```suggestion
                call az storage blob sync --container $web --account-name pagopadcheckoutsa -s "$(Pipeline.Workspace)\Bundle_DEV" --exclude-path payment-transactions-gateway;ecommerce-fe
```
looking at [azcopy docs](https://learn.microsoft.com/en-us/azure/storage/common/storage-ref-azcopy-copy) multiple paths have to be separated by semicolon

**Review by pietro-tota — CHANGES_REQUESTED** [review]
azcopy --exclude-path paths should be separated by semicolon, right?

**Review by pietro-tota — APPROVED** [review]

File: .devops/pagopa-deploy-pipelines.yml (hunk 1 above)
Comment (ciuffagianluca) [reply .devops/pagopa-deploy-pipelines.yml]: Set in this way while you were posting :)

**Comment by ciuffagianluca** [comment]
Bypass branch protection for integration test failed as known in main branch

**Pull request merged**